docs/streams/developer-guide.html

```diff
@@ -267,21 +295,532 @@
 
     <h4><a id="streams_dsl_source" href="#streams_dsl_source">Create Source Streams from Kafka</a></h4>
 
-    <p>
-        Either a <b>record stream</b> (defined as <code>KStream</code>) or a <b>changelog stream</b> (defined as <code>KTable</code> or <code>GlobalKTable</code>)
-        can be created as a source stream from one or more Kafka topics (for <code>KTable</code> and <code>GlobalKTable</code> you can only create the source stream
-        from a single topic).
-    </p>
-
-    <pre class="brush: java;">
-    KStreamBuilder builder = new KStreamBuilder();
+    <table class="data-table" border="1">
+  <tbody><tr>
+          <th>Reading from Kafka</th>
+          <th>Description</th>
+  </tr>
+      <tr>
+        <td><b>Stream</b>: input topic(s) -> <code>KStream</code></td>
+        <td>Create a <code>KStream</code> from the specified Kafka input topic(s), interpreting the data as a record stream.
```

Comment:
Ideally, we'd linkify "KStream" here and point it to the appropriate section in the (soon updated) Concepts page at https://kafka.apache.org/0110/documentation/streams/core-concepts, right?  (same comment applies to other sections in this docs table, e.g. for "KTable")